.changeset/unlucky-carpets-dance.md

```diff
@@ -0,0 +1,8 @@
+---
+"@frameless/pdc-dashboard": minor
+"@frameless/vth-dashboard": minor
+"@frameless/strapi-tiptap-editor": minor
+---
+
+- Toegevoegd: Nederlandse vertalingen voor TipTap editor extensies. Bij selectie van de Nederlandse taal in het Strapi-dashboard wordt de toolbar in het Nederlands weergegeven.
+- Toegevoegd: Focusfunctionaliteit voor de editor table-extensie in het Strapi-dashboard. Bij het invoegen van een tabel kan de muiscursor eenvoudig boven en onder de tabel worden verplaatst. Bij hover wordt een border en twee invoegknoppen weergegeven. Meer details in [de handleiding](linkhttps://vth-api.cg-intern.acc.utrecht.nl/manual/index.html).
```

Comment:
In plaats van een unordered list gebruiken, zou ik deze opsplitsen in twee files: dan krijg je mooi twee extra items in de changelog, in plaats van een nested ordered list.